arch/arm/src/imxrt/imxrt_flexio.c

```diff
@@ -788,35 +788,43 @@ static uint32_t imxrt_flexio_get_shifter_buffer_address(
   switch (type)
     {
     case FLEXIO_SHIFTER_BUFFER:
-      address = (uint32_t)(IMXRT_FLEXIO_SHIFTBUF0_OFFSET + index * 0x4);
```

Comment:
The function description is inconsistent. One part states that the function returns the buffer address and another part that it returns the buffer index.

Reply:
> The function description is inconsistent. One part states that the function returns the buffer address and another part that it returns the buffer index.

Thanks. Fixed.